### PR TITLE
Use Golden Image for Preview Instances

### PR DESCRIPTION
**Pull request by cthulhuplus**
**This PR is no longer relevant, the changes have been merged in as a part of other work and is being left open to provide a safe place for the new team members to explore.**

Changes to Previews deploys to use the Golden AMI.

### This pull request changes...

- Uses latest golden image
- For the application to function as if there had been no change

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by cthulhuplus** [comment]
I was able to log in to the app, but the OS that the app is built upon is different, so I'd like everyone to really have at this preview to make sure there aren't undiscovered flaws before we merge this change in.

**Comment by beparticular** [comment]
@cthulhuplus are there any anticipated flaws, or major areas of concern? i'm not seeing anything yet, but want to make sure I'm looking in the right places.

**Comment by cthulhuplus** [comment]
> @cthulhuplus are there any anticipated flaws, or major areas of concern? i'm not seeing anything yet, but want to make sure I'm looking in the right places.

@beparticular I am hoping that I caught everything. I invited everyone to review it because I want every one to poke at it in their own way. So, just use it how you would and if you find anything let me know. Hopefully you wont!

**Review by thetif — APPROVED** [review]
looks good from what I understand about these aws scripts

**Review by beparticular — APPROVED** [review]

File: bin/preview-deploy/aws.user-data.sh [hunk 1]
```diff
@@ -96,7 +100,7 @@ http {
     }
 }
 NGINXCONFIG
-service nginx restart
+#service nginx restart
```
Comment (radavis) [inline bin/preview-deploy/aws.user-data.sh:103]: Delete this line.

File: bin/preview-deploy/aws.user-data.sh [hunk 2]
```diff
@@ -14,8 +14,9 @@ chmod -R g+w /app
 mkdir /app/tls
 
 # Install nginx and postgres
-amazon-linux-extras install nginx1.12
-yum -y install git postgresql-server amazon-cloudwatch-agent
+#amazon-linux-extras install nginx1.12
```
Comment (radavis) [inline bin/preview-deploy/aws.user-data.sh:17]: Delete this line.

**Review by radavis — CHANGES_REQUESTED** [review]
Looks good other than containing some commented code.

https://kentcdodds.com/blog/please-dont-commit-commented-out-code

**Review by tbolt — COMMENTED** [review]
Seems OK to me. Is there something specific I can test?

**Comment by cthulhuplus** [comment]
This PR is no longer relevant, the changes have been merged in as a part of other work and is being left open to provide a safe place for the new team members to explore.

**Pull request closed without merging**